### DISCUSSION TITLE
Stream only audio

### DISCUSSION THREAD

**Discussion by Mintakaaaa** [PierfrancescoSoffritti/android-youtube-player#1095] [General]
Hi, thanks so much for this repository, it's great!
Is it possible to only stream audio instead of both audio and video using this library?
Currently, when i load a video using loadVideo function, it loads audio and video. I'd like to just load audio if that's possible.

**Comment by PierfrancescoSoffritti** [comment]
Hi, it is not possible unfortunately. It's against YouTube terms of service.

**Reply by Mintakaaaa** [reply]
I see. I was asking because I'd like to make an app where I can play youtube videos in the background (while phone is locked) without streaming the video if the phone is locked since that uses up more bandwidth than would be used if I just streamed audio only.
Could you confirm if this is already the case with your library?

**Reply by PierfrancescoSoffritti** [reply]
You can play a video in the background, but it break YouTube terms of services, so you won't be able to publish the app.

**Reply by Mintakaaaa** [reply]
That's ok, I don't want to publish the app anyway, I just want to use it for myself. My question was about bandwidth usage though.

**Reply by PierfrancescoSoffritti** [reply]
All data would be streamed, video and audio.

**Reply by Mintakaaaa** [reply]
And that applies even when the phone is locked, right?

**Reply by PierfrancescoSoffritti** [reply]
Yes.

**Reply by Mintakaaaa** [reply]
Alright, thank you.
